packages/user/integration-tests/__fixtures__/event-bus.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  EmitData,
+  EventBusTypes,
+  IEventBusModuleService,
+  Message,
+  Subscriber,
+} from "@medusajs/types"
+
+export class EventBusService implements IEventBusModuleService {
```

Comment:
nit(non-blocking): Could be moved to be general-purpose. I imagine the same will be needed for all modules